asciidoc/volume2/gateways/tf2-ch-b-gateway-dec.adoc

```diff
@@ -287,77 +287,92 @@ a| *SDPi Supplement Version Note*:  For SDPi 1.0 DEV Internal Review Version, th
 
 |===
 
+[#ref_dec_OBR7_mapping]
 ====== OBR-7 Observation Date/Time
 The OBR-7 field specifies the time point or start of time interval for all OBX segments within the scope of this OBR segment, that is, OBX segments that are part of the ORDER_OBSERVATION segment group, that do not specify an overriding time point in the OBX-14 field.
 
 The presence of an overriding time point in OBX-14 indicates an episodic measurement such as non-invasive blood pressure.
 
 The absence of an overriding time point in the OBX-14 field implies that this is an instance of a periodically sampled observation with a time stamp given by OBR-7 field.
 
-
-.R8006
-[sdpi_requirement#r8006,sdpi_req_level=shall]
+.R8037
+[sdpi_requirement#r8037,sdpi_req_level=shall]
 ****
 
-A <<actor_somds_dec_gateway>> shall set the OBR-7 field to the date and time of all continuously (periodically) measured metrics reported by a *msg:PeriodicMetricReport* to be exported within the same *msg:RetrievabilityInfo/@UpdatePeriod*.
+A <<actor_somds_dec_gateway>> shall export continuously (periodically) measured metrics periodically at a defined interval.
 
 .Notes
 [%collapsible]
 ====
-NOTE: If, for example, the *@UpdatePeriod* is set to one second, a *msg:PeriodicMetricReport* is expected every second with new periodic metric states to be exported within this one second interval. The OBR-7 field is set to the start time of the interval. The individual periodic metric *@DeterminationTime* is basically ignored, but has to be within the time boundaries of the current export interval.
+NOTE: It is up to the <<actor_somds_dec_gateway>> how the export interval is defined. The interval might be a fixed interval of e.g. 30 seconds, or a configurable interval ranging e.g. between 10 seconds and 2 minutes.
 
-NOTE: <<ref_tbl_dec_obr7_mapping>> defines the mapping of the SDC metric measurement timestamp to the data fields of the HL7 data type *DTM* used in the OBR-7 field.
 ====
 ****
 
-.R8007
-[sdpi_requirement#r8007,sdpi_req_level=shall]
+.R8038
+[sdpi_requirement#r8038,sdpi_req_level=shall]
 ****
 
-For episodic measurements and the absence of any continuously measured metrics for this export interval, a <<actor_somds_dec_gateway>> shall set the OBR-7 field to the oldest measurement timestamp of the episodic metrics to be exported in the current export interval.
+A <<actor_somds_dec_gateway>> shall set the OBR-7 field to the start date and time of current export interval.
```

Comment:
Proposed extension/clarification:
"For continuously (periodically) measured metrics, a SOMDS DEC Gateway shall [...]"

Reply:
The start date and time of the current export interval is independent of the metric measurement type. The interval is defined by the gatewasy and also the eperiodic data are exported in the interval but with the measurment time set in the OBX-14 field.